otx/mpa/cls/incremental/stage.py

```diff
@@ -35,35 +35,27 @@ def configure_task(self, cfg, training, **kwargs):
     # noqa: C901
     def configure_task_adapt(self, cfg, training, **kwargs):
         """Configure for Task Adaptation Task"""
-
-        self.adapt_type = cfg["task_adapt"].get("op", "REPLACE")
-        train_data_cfg = Stage.get_data_cfg(cfg, "train")
+        train_data_cfg = self.get_data_cfg(cfg, "train")
         if training:
             if train_data_cfg.type not in CLASS_INC_DATASET:
                 logger.warning(f"Class Incremental Learning for {train_data_cfg.type} is not yet supported!")
             if "new_classes" not in train_data_cfg:
                 logger.warning('"new_classes" should be defined for incremental learning w/ current model.')
 
             if cfg.model.type in WEIGHT_MIX_CLASSIFIER:
+                cfg.task_adapt.final = self.model_classes
```

Comment:
I think `cfg.task_adapt.final` isn't used anywhere. How about deleting this line?

Reply:
Done